Build.cmd

```diff
@@ -43,6 +43,11 @@ call mvn.cmd clean
 @rem build the package
 call mvn.cmd package
 
+if %ERRORLEVEL% NEQ 0 (
+	@echo Build SparkCLR Scala components failed, stop building.
+	popd
+	goto :eof
```

Comment:
I think you also want to ensure than this script [here and below] returns a non-zero result code if a build step fails. 

Compare with the `goto :ErrorStop` and `exit /b %RC%` code in the "other" Build.cmd script.
https://github.com/Microsoft/SparkCLR/blob/master/csharp/Build.cmd#L67